src/main/java/ch/njol/skript/effects/EffCopy.java

```diff
@@ -0,0 +1,111 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.Variable;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.Map;
+
+@Name("Copy")
+@Description("Copies objects into a variable. When copying a list over to another list, the source list and its sublists are also copied over.")
+@Examples({
+	"set {_foo::bar} to 1",
+	"set {_foo::sublist::foobar} to \"hey\"",
+	"copy {_foo::*} to {_copy::*}",
+	"broadcast indices of {_copy::*} # bar, sublist",
+	"broadcast {_copy::bar} # 1",
+	"broadcast {_copy::sublist::foobar} # \"hey!\""
+})
+@Since("INSERT VERSION")
+@Keywords({"copy", "clone"})
```

Comment:
Copy is already a keyword due to element name
```suggestion
@Keywords({"variables", "clone"})
```